website/docs/secrets/manage-secrets-gui.mdx

```diff
@@ -3,18 +3,131 @@ title: Manage Secrets GUI
+
+- **Status:** This indicates the current status of the external secret, which can be "Ready" or "Not Ready" depending on whether the external secret has been successfully created and is ready for use.
+
+- **Last Updated:** This shows the date and time when the external secret was last updated.
+
+- **External Secret:** This is the name of the external secret that you are viewing.
+
+- **K8s Secret:** This is the name of the Kubernetes secret that is associated with the external secret.
+
+- **Cluster:** This indicates whether the external secret is scoped to a specific namespace or to the entire cluster.
+
+- **Secret Store:** This shows the name of the secret store provider that is being used to store the external secret.
+
+- **Secret Store Type:** This indicates the type of secret store that is being used to store the external secret. In this case, the type is "AWS Secrets Manager".
+
+- **Secret path:** This is the path to the external secret within the cluster.
+
+- **Property:** This is the property or key that is associated with the external secret.
+
+- **Version:** This shows the version of the external secret, which may be blank if no version has been specified.
+
+Understanding the information provided on the External Secret Details page can help you to manage and troubleshoot your external secrets as needed.
+
+
+### Understanding Events
+
+![External Secret Events](./imgs/external-secret-events-1.png)
+
+The following events can be expected when using the Manage Secrets GUI to manage external secrets:
+
+- **Updated:** This event indicates that an existing external secret has been successfully updated with new data.
+
+- **Not Ready:** This event indicates that there was an issue with the secret store when trying to access or synchronize external secrets. This includes situations such as the secret store being unavailable or not ready to handle requests, or issues with authentication when trying to access the secret store.
+
+Understanding these events can help you to troubleshoot issues that may arise when managing external secrets using the Manage Secrets GUI. In particular, if you encounter a "Not Ready" event, you may need to check your secret store credentials and ensure that the secret store is operational before proceeding with any further actions.
+
+### Sync External Secrets
+
+Once you have created an External Secret, it is important to keep it up to date with the latest changes in the corresponding external secret manager. This can be achieved by syncing the External Secret with its source using the Sync feature provided by WeaveWorks Enterprise.
+
+To sync an External Secret, navigate to the List External Secrets page and select the External Secret you want to sync. Once you are on the External Secret Details page, click on the Sync button on the top left corner of the page. This will trigger a sync operation for the selected External Secret.
```

Comment:
```suggestion
To sync an External Secret, navigate to the Secrets page and select the External Secret you want to sync. Once you are in the details page, click on the Sync button on the top left corner of the page. This will trigger a sync operation for the selected External Secret.
```